docs(community): improve documentation for v2 community release (#706)
novuhq/docs: community/self-hosting-novu/deploy-with-docker.mdx

  These env variables should be present on all services novu provides due to tight
  coupling.
</Note>

## FAQs
### Local Tunnel and Self-Hosted Deployments

> Local tunnel support is not included as part of the Novu Docker images. This is typically unnecessary for many users, depending on how their environment is structured.

Reviewer: Please prepend the following first sentence: "Novu uses a local tunnel to connect to Local Studio during development."

  These env variables should be present on all services novu provides due to tight
  coupling.
</Note>

> Here’s an improved version of your write-up with additional details and clarification:

Reviewer: Please remove

Reviewer: Should we maybe call this section `FAQ`? I feel it needs a title.

Author: Removed. I was also confused with title. I did not keep it FAQs because I only talked about Local Tunnel and Self-Hosted Deployments. Let me rename and reorganise

---

## Local Tunnel and Self-Hosted Deployments

Local tunnel support is not included as part of the Novu Docker images. This is typically unnecessary for many users, depending on how their environment is structured.

### When is Local Tunnel Not Required?
If the customer’s application and the self-hosted Novu deployment are within the same network, there is no need for a local tunnel. In this case, the application can communicate directly with Novu through the internal network.

### When is Local Tunnel Required?

> If the application and Novu deployment reside on different networks, you can still interact with your self-hosted Novu instance using the Novu CLI without requiring a local tunnel. The CLI allows you to specify the Dashboard URL and Bridge Endpoint Origin to enable communication across networks. 

Reviewer: ```suggestion
If the application and Novu deployment reside on different networks, you can still interact with your self-hosted Novu instance using the Novu CLI. The CLI allows you to specify the Dashboard URL and Bridge Endpoint Origin to enable communication across networks via the Novu Cloud Local Tunnel. 
```
